client/src/components/ImageCard/ImageCard.css

```diff
@@ -0,0 +1,35 @@
+.backgroundImage {
+  position: relative;
+  height: 100%;
+  width: 100%;
+}
+
+.textHolder {
+  position: absolute;
+  top: 12%;
+  left: 0;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  height: 70%;
+  align-items: center;
+}
```

Comment:
Hey, do we need to use absolute positioning here? Could we apply the image as a background image and then just align the text with flexbox